src/main/java/com/stripe/net/StripeRequest.java

```diff
@@ -71,7 +71,52 @@ public StripeRequest(
       this.options = (options != null) ? options : RequestOptions.getDefault();
       this.method = method;
       this.url = buildURL(method, url, params);
-      this.content = buildContent(method, params);
+
+      if (this.options instanceof RawRequestOptions) {
+        RawRequestOptions rawOptions = (RawRequestOptions) this.options;
+        this.content = buildContent(method, params, rawOptions.getApiMode());
+      } else {
+        this.content = buildContent(method, params, RawRequestOptions.ApiMode.STANDARD);
+      }
+
+      this.headers = buildHeaders(method, this.options);
+    } catch (IOException e) {
```

Comment:
unrelated but how can this exception even happen in the ctor?

Reply:
`buildURL` and `buildContent` throw `IOException`'s